tests/unit/test_table.py

```diff
@@ -1735,8 +1901,25 @@ def test_do_mutate_retryable_rows_second_retry(self):
         response = self._make_responses([self.SUCCESS, self.RETRYABLE_1])
 
         # Patch the stub used by the API method.
-        inner_api_calls = client._table_data_client._inner_api_calls
-        inner_api_calls["mutate_rows"] = mock.Mock(side_effect=[[response]])
+        client._table_data_client.mutate_rows.side_effect = [[response]]
+        data_api.table_path.return_value = (
+            "projects/"
```

Comment:
f-str. Also, as much as this is repeated, could we make this a helper?

Reply:
A lot of these were unnecessary leftover from when I was figuring out what to properly mock. Cleaning up.